src/components/StatePicker/index.js

```diff
@@ -0,0 +1,93 @@
+import React, {useEffect, useState} from 'react';
+import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import MenuItemWithTopDescription from '../MenuItemWithTopDescription';
+import useLocalize from '../../hooks/useLocalize';
+import FormHelpMessage from '../FormHelpMessage';
+import StateSelectorModal from './StateSelectorModal';
+
+const propTypes = {
+    /** Error text to display */
+    errorText: PropTypes.string,
+
+    /** State to display */
+    value: PropTypes.string,
+
+    /** Callback to call when the input changes */
+    onInputChange: PropTypes.func,
+
+    /** A ref to forward to MenuItemWithTopDescription */
+    forwardedRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]),
```

Comment:
When we migrated the `StatePicker` component, we missed adding the `label` prop, which caused a regression [here](https://github.com/Expensify/App/issues/23870).